pkg/northbound/gnmi/set.go

```diff
@@ -155,35 +179,35 @@ func (s *Server) Set(ctx context.Context, req *gnmi.SetRequest) (*gnmi.SetRespon
 	for k, v := range targetRemoves {
 		targetRemovesCopy[k] = v
 	}
-	updateResultsOld, networkChanges, err :=
-		s.executeSetConfig(targetUpdatesCopy, targetRemovesCopy, string(version), string(deviceType), netCfgChangeName)
+	//updateResultsOld, networkChanges, err :=
```

Comment:
Can we remove this commented out block now?

Reply:
we'll leave it in until cleanup later